probe.go

```diff
@@ -203,6 +210,9 @@ type Probe struct {
 	// enabled, this is max(10, 2*NR_CPUS); otherwise, it is NR_CPUS. For kprobes, maxactive is ignored.
 	KProbeMaxActive int
 
+	// DefaultKprobeAttachMethod: Method to use for attaching the kprobe. Either use perfEventOpen ABI or kprobe events
+	DefaultKprobeAttachMethod KprobeAttachMethod
```

Comment:
Nit: I think it would make things clearer if we simply called it `KprobeAttachMethod`.
Nit Nit: `// DefaultKprobeAttachMethod: Method to use for attaching ...` => `// DefaultKprobeAttachMethod - Method to use for attaching ...`